### PR TITLE
Write lab

### PR DESCRIPTION
**Pull request by kcleland0818**
This change is being made in the consumer canvas per the conversation on 7/22 to keep the same modules for Blackrock and AWS to streamline.

Let me know if you think this is too much work for a lab.

Lab solution here: [Spring Mod 1 Data Lab](https://docs.google.com/document/d/1_Y2zL83C1lwbGWT4QuPrx6oxAGw4Rx3NJ8ky6Jbusrc/edit)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,30 +2,287 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Practice creating a web application that connects to a data source using the
+  Spring Framework.
+- Connect an application to a PostgreSQL database.
+- Create a DTO and `@Entity` class.
+- Create a `@Repository` class.
```
Comment (developerjaym) [inline README.md]: Consider changing 'class' to 'interface'

File: src/main/resources/lab_setup.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
+DROP TABLE IF EXISTS member;
```
Comment (developerjaym) [inline src/main/resources/lab_setup.sql]: Could/should we configure the Spring Boot app to run this file on start up?

File: README.md (hunk 1 above)
Comment (kcleland0818) [reply README.md]: Yes this is a typo. Thanks!

File: src/main/resources/lab_setup.sql (hunk 2 above)
Comment (kcleland0818) [reply src/main/resources/lab_setup.sql]: Hmm... I could change it to data.sql to have it run on start up to eliminate some pre-work! Good idea!

**Review by developerjaym — APPROVED** [review]

**Review by linda-seiter — APPROVED** [review]

**Pull request merged**